### PR TITLE
fix formatting

### PR DESCRIPTION
**Pull request by mortendahl**

### CODE REVIEW

File: primitives/tf_encrypted/primitives/sodium/python/easy_box_test.py [hunk 1]
```diff
@@ -1,12 +1,12 @@
 # pylint: disable=missing-docstring
 import unittest
-from absl.testing import parameterized
 
 import numpy as np
 import tensorflow as tf
+from absl.testing import parameterized
```
Comment (jvmncs) [inline primitives/tf_encrypted/primitives/sodium/python/easy_box_test.py:6]: do we have an issue about adding `isort`? that'd be good to have

Comment (mortendahl) [reply primitives/tf_encrypted/primitives/sodium/python/easy_box_test.py:6]: what do you mean by "adding"?

just realized that adding `flake8-isort` works, so did that.

Comment (jvmncs) [reply primitives/tf_encrypted/primitives/sodium/python/easy_box_test.py:6]: meant adding it to CI, which it looks like you've done in Makefile here

Comment (mortendahl) [reply primitives/tf_encrypted/primitives/sodium/python/easy_box_test.py:6]: 👍

**Pull request merged**